python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -21,19 +21,23 @@
 
 
 def assert_arg_type_is_supported(ty):
-    SUPPORTED = [np.float32, np.float64, np.int32, np.int64, np.bool_]
+    SUPPORTED = [np.float32, np.float64, np.uint8, np.int8, np.int32, np.int64, np.bool_]
     assert ty in SUPPORTED, f"Only numpy arrays with dtypes in {SUPPORTED} are supported"
 
 
 memref_type_to_np_dtype = {
     "mrf32": np.float32,
     "mrf64": np.float64,
     "mri1": np.bool_,
+    "mrui8": np.uint8,
+    "mri8": np.int8,
     "mri32": np.int32,
     "mri64": np.int64
 }
 elemental_type_to_ctype = {
     "i1": ctypes.c_bool,
+    "ui8": ctypes.c_ubyte,
```

Comment:
I don't think `ui8` is a type that exists in MLIR

Reply:
Yes, at this level, ui8 should not exist.